src/httprpc.cpp

```diff
@@ -63,6 +63,8 @@ class HTTPRPCTimerInterface : public RPCTimerInterface
 static std::string strRPCUserColonPass;
 /* Stored RPC timer interface (for unregistration) */
 static std::unique_ptr<HTTPRPCTimerInterface> httpRPCTimerInterface;
+/* RPC Auth Whitelist */
+static std::map<std::string, std::set<std::string>> whitelistedRPC;
```

Comment:
`rpc_whitelist`?

Reply:
style-nit in commit b411ae6500: `g_rpc_whitelist`, because it is a global